008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1021 @@
+
+To achieve this, we update the behavioural weights as follows:
+
+\begin{itemize}
+    \item If the agent's health is less than 20:
+    \begin{enumerate}
+        \item \textit{weightHP} $\leftarrow$ \textit{weightHP} + 0.05
+        \item \textit{weightFloor} $\leftarrow$ \textit{weightFloor} - 0.05
+    \end{enumerate}
+    \item If the agent's average food intake is less than 1:
+    \begin{enumerate}
+        \item \textit{weightHP} $\leftarrow$ \textit{weightHP} - 0.1
+        \item \textit{weightFloor} $\leftarrow$ \textit{weightFloor} + 0.1
+    \end{enumerate}
+\end{itemize}
+
+In addition, we use sufficient boundary conditions to ensure that the individual weights consistently fall within the range [0,1].
+
+\subsubsection{Behavioural Update}
+
+Based on the mapping of social motives from 0.0 to 10.0, a lower floor (\emph{i.e.,} greater floor number) and a lower HP (\emph{i.e.,} smaller HP value) push the social motive variable to be higher. The weights are hence chosen to reflect these correlations (initialised as 0.3 resp. 0.7), along with the relevant feature transformations (\Cref{sec:featTran}) applied to these factors.
+
+This weighted sum represents the prediction of the agent's next social motive, titled \textit{nextBehaviourPrediction} in \Cref{fig:socialMotives}. We therefore calculate:
+
+\begin{equation}
+    \mathit{behaviourUpdate = nextBehaviourPrediction - currentBehaviour}
+\end{equation}
+
+
+For the first iteration, \textit{currentBehaviour} = \textit{baseBehaviour} as presented in \Cref{fig:socialMotives}.
+
+\subsection{Willingness to Change}
+Although the above encapsulates an essential human characteristic, the ability to change, another important element, the willingness to change, is missing. Through this next layer of complexity, we attempt to quantify how resistant an agent is to altering its behaviour. This concept brings forth more freedom for modelling different human characteristics and realistic situations. For example, it is now possible to distinguish between an agent that is initially an altruist, but may then become selfish as the supply of food diminishes, and an agent that is a “true altruist”, which retains the same social motive throughout. Similarly, this is extended to the distinction between a true narcissist agent that never changes, and a narcissist agent that may, during the game, witness others' suffering whilst itself has an abundance of food and therefore steer towards a more collectivist behaviour.
+
+To represent the willingness of an agent to change its social motive, an additional parameter, labelled as \textit{stubbornness}, is introduced and instantiated as a number between 0.0 and 1.0. This number (or, precisely, one minus this number) is multiplied with the behaviour update. This operation is similar to a step-size, and reflects how easy it is for the social motive to change. We therefore calculate the \textit{scaledUpdate} and \textit{newBehaviour} as shown below.
```

Comment:
```suggestion
To represent the willingness of an agent to change its social motive, an additional parameter, labelled as \textit{stubbornness}, is introduced and instantiated as a number between 0.0 and 1.0. This number is used to scale the behaviour update. This operation is similar to a step-size, and reflects how easy it is for the social motive to change. We therefore calculate the \textit{scaledUpdate} and \textit{newBehaviour} as shown below.
```

Reply:
'This number (or, precisely, one minus this number)' doesn't make sense - it's either one or the other.